src/test/java/fr/insee/pearljam/infrastructure/surveyunit/entity/CommunicationRequestDBTest.java

```diff
@@ -1,46 +1,47 @@
 package fr.insee.pearljam.infrastructure.surveyunit.entity;
 
 import fr.insee.pearljam.api.domain.SurveyUnit;
-import fr.insee.pearljam.domain.surveyunit.model.communication.*;
-import fr.insee.pearljam.infrastructure.campaign.entity.CommunicationTemplateDB;
-import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.DisplayName;
-import org.junit.jupiter.api.Test;
-
+import fr.insee.pearljam.domain.surveyunit.model.communication.CommunicationRequest;
+import fr.insee.pearljam.domain.surveyunit.model.communication.CommunicationRequestEmitter;
+import fr.insee.pearljam.domain.surveyunit.model.communication.CommunicationRequestReason;
+import fr.insee.pearljam.domain.surveyunit.model.communication.CommunicationRequestStatus;
+import fr.insee.pearljam.domain.surveyunit.model.communication.CommunicationStatusType;
+import fr.insee.pearljam.infrastructure.campaign.entity.CommunicationTemplateDBId;
 import java.util.List;
-
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.tuple;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
 
 class CommunicationRequestDBTest {
 
     private SurveyUnit surveyUnit;
-    private CommunicationTemplateDB communicationTemplate;
 
     @BeforeEach
     void setup() {
         surveyUnit = new SurveyUnit();
         surveyUnit.setId("su-id");
-        communicationTemplate = new CommunicationTemplateDB();
-        communicationTemplate.setId(1L);
+
```

Comment:
gonna need it back, as communicationRequest fromModel will use CommunicationTemplate to get CommunicationTemplateDbId